submissions/MutmainahOdunewu/HTML/index.html

```diff
@@ -1,99 +1,141 @@
+          <div class="card">
+            <img src="images/sneakers.jpeg" />
+            <p>Sneakers</p>
+            <p>Price:#15000</p>
           </div>
-          <div class="foot">
-               <p>CONNECT WITH US</p>
-               <ul id="socials">
-                  <li><a href="#"><img src="./images/facebooklogo.png"></a></li>
-                  <li><a href="#"><img src="./images/twitterlogo.png"></a></li>
-                  <li><a href="#"><img src="./images/gmaillogo.png" ></a></li>
-                  <li><a href="#"><img src="./images/youtubelogo.png"></a></li>
-               </ul>
-         </div>
-       </div>  
-       <p id="copyright">&copy; Copyright 2020 sandalsbymo.com <span> All rights reserved</span></p>
-      </footer>
-</body>
-</html>
\ No newline at end of file
+          <div class="card">
+            <img src="images/Halfshoe.jpg" />
+            <p>Halfshoe</p>
+            <p>Price:#10000</p>
+          </div>
+        </div>
+      </section>
+    </div>
+    <footer>
+      <div class="info">
+        <div class="foot">
+          <ul id="quick-links">
+            <li><a href="#">About Us</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Terms and Conditions</a></li>
+          </ul>
+        </div>
+        <div class="foot">
+          <p>Sign up for our Newsletters</p>
+          <form>
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="Enter your email address"
+            />
+            <button type="submit" class="styling">Submit</button>
+          </form>
+        </div>
+        <div class="foot">
+          <p>CONNECT WITH US</p>
+          <ul id="socials">
+            <li>
+              <a href="#"><img src="./images/facebooklogo.png" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/twitterlogo.png" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/gmaillogo.png" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./images/youtubelogo.png" /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <p id="copyright">
+        &copy; Copyright 2020 sandalsbymo.com <span> All rights reserved</span>
+      </p>
+    </footer>
+    <script>
+      const productList = [
+        {
+          productName: "Boots",
+          imgUrl: "images/Boots.jpg",
+          productPrice: "#20000",
+        },
+        {
+          productName: "Sneakers",
+          imgUrl: "images/sneakers.jpg",
+          productPrice: "#15000",
+        },
+        {
+          productName: "Halfshoe",
+          imgUrl: "images/Halfshoe.jpg",
+          productPrice: "#10000",
+        },
+      ];
+      for (let i of productList) {
+        const output = `${i.productName} with ${i.imgUrl} is sold for ${i.productPrice}`;
+        console.log(output);
+      }
+    </script>
```

Comment:
Your code doesn't look clean this way. Kindly create a seperate .js file for the script, you can then place the script src in the html file.